en/docs/learn/configuring-oidc-federated-idp-initiated-logout.md

```diff
@@ -0,0 +1,198 @@
+
+    ![oidc-fed-idp-config-in-primary-idp](../assets/img/tutorials/oidc-fed-idp-config-in-primary-idp.png)
+
+8. Signature of the logout token is validated using either the registered JWKS uri or uploaded certificate to the
+   relevant identity provider. 
+   
+    - Under the **Basic Information** section, select the **Use IDP JWKS endpoint** option from **Choose IDP certificate
+     type** and add the JWKS uri
+     `https://localhost.com:9444/oauth2/jwks` to Identity Provider's JWKS Endpoint.
+    ![oidc-primary-idp-jwks-uri-config](../assets/img/tutorials/oidc-primary-idp-jwks-uri-config.png)
+      
+    - Alternatively, select the **Upload IDP certificate** option from **Choose IDP certificate type** and upload the
+     certificate of the SecondaryIS.
+    ![oidc-primary-idp-certificate-config](../assets/img/tutorials/oidc-primary-idp-certificate-config.png)
+     
+
+9. Click **Register**.
+
+### Configuring Pickup Dispatch application in the Primary IS
+
+1. Follow the steps
+   in [Deploying the pickup-dispatch webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-pickup-dispatch-webapp)
+   to download, deploy and register the **Pickup-Dispatch** sample.
+2. Once you have added the OIDC service provider, go to **Service Provider Configuration** and expand **Local & Outbound
+   Authentication Configuration**.
+3. Select **Federated Authentication** and from the dropdown menu select **SecondaryIS**. Click **Update**.
+
+   ![oidc-service-provider-federated-authentication](../assets/img/tutorials/oidc-service-provider-federated-authentication.png)
+
+### Configuring Pickup Manager application in the Secondary IS
+
+Follow the steps
+in [Deploying the pickup-manager webapp](https://is.docs.wso2.com/en/latest/learn/deploying-the-sample-app/#deploying-the-pickup-manager-webapp)
+to download, deploy and register the Pickup-Manager sample.
+
+## OIDC Back-channel Logout Token Validation
+
+The following is an example OIDC back-channel logout token.
+
+```
+   {
+   "iss": "https://localhost.com:9444/oauth2/token",
+   "sub": "admin",
+   "aud": "w_Hwp05dFRwcRs_WFHv9SNwpflAa",
+   "iat": 1609911868,
+   "exp": 1609911988,
+   "jti": "16159e3e-c5fc-42de-b93f-b0782ab33d58",
+   "sid": "15043ffc-877d-4205-af41-9b107f7da38c",
+   "events": {
+     "http://schemas.openid.net/event/backchannel-logout": {}
+     }
+```
+
+Logout token validation is done according to
+the [OIDC back-channel logout specification](https://openid.net/specs/openid-connect-backchannel-1_0.html#Validation)
+for the token signature and
+`iss`, `aud`, `iat`, `sub`, `sid`, `events` and `nonce` claims.
+
+### Configure “iat” claim validation
+
+To disable or change the “iat” claim validation, the following configuration needs to be added to the **deployment.
+toml**
+file in `<PRIMARY_IS_HOME>/repository/conf/`
+
+``` 
+[authentication.authenticator.oidc.parameters] 
+enableIatValidation = true 
```

Comment:
Can we add configuration separately to how to disable and how to change the iat validation.

To disable it,

[authentication.authenticator.oidc.parameters] 
enableIatValidation = false 

[authentication.authenticator.oidc.parameters] 
iatValidityPeriod = "150" 
Also menntion that this is enabled by default and mwntion the default iatvalidation time period